Source/Concurrency/YieldingProcInstrumentation.cs

```diff
@@ -545,7 +541,9 @@ private void DesugarConcurrency(Implementation impl, List<Cmd> preconditions)
       impl.Blocks.Add(unchangedCheckerBlock);
       impl.Blocks.Add(returnCheckerBlock);
       impl.Blocks.Add(returnBlock);
-      impl.Blocks.AddRange(implRefinementCheckingBlocks);
+      foreach (var block in implRefinementCheckingBlocks) {
+        impl.Blocks.Add(block);
+      }
```

Comment:
Why is this preferred?

Reply:
`AddRange` does not exist on `IList`

Reply:
Ah damn, feels like a ripe opportunity for an extension method :P